src/redux/actions/RocketsActions.js

```diff
@@ -0,0 +1,28 @@
+export const LOAD_ROCKETS = 'LOAD_ROCKETS';
+
+export const loadRockets = (payload) => ({
+  type: LOAD_ROCKETS,
+  payload,
+});
+
+async function fetchData() {
+  const response = await fetch('https://api.spacexdata.com/v3/rockets');
+  const data = await response.json();
+  return data;
+}
+
+export const displayRockets = () => async (dispatch) => {
+  const rocketsArr = await fetchData();
+  const rocketsTemp = [];
+
+  Object.keys(rocketsArr).forEach((id) => {
```

Comment:
FYI, the "Object" prototype has an `Object.entries` which may prove useful in the future.
Here's a [resource](https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Object/entries) if you want to learn about it.